pkg/kubectl/cmd/resource/get_test.go

```diff
@@ -27,6 +27,8 @@ import (
 	"strings"
 	"testing"
 
+	"github.com/stretchr/testify/assert"
```

Comment:
I don't think we should import this

Reply:
OK. I'll change to `t.Fatalf` function soon :ok_man:

Reply:
@shiywang Modified. 😄 PTAL